### PR TITLE
Update hashes to the latest versions of the dependencies

### PR DESCRIPTION
**Pull request by MarcFontaine**
Update the hashes and make the changes to fix the build.

### CODE REVIEW

File: cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx/NodeToNode.hs [hunk 1]
```diff
@@ -220,15 +220,15 @@ benchmarkConnectTxSubmit iocp trs cfg localAddr remoteAddr myTxSubClient =
  where
   myCodecs :: Codecs blk DeserialiseFailure m
                 ByteString ByteString ByteString ByteString ByteString
-  myCodecs  = defaultCodecs (getCodecConfig $ configBlock cfg) (mostRecentNodeToNodeVersion (Proxy @blk))
```
Comment (MarcFontaine) [inline cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx/NodeToNode.hs:223]: much better

**Comment by CodiePP** [comment]
we are waiting for https://github.com/input-output-hk/cardano-node/pull/1288 being merged.

**Comment by CodiePP** [comment]
> we are waiting for [input-output-hk/cardano-node#1288](https://github.com/input-output-hk/cardano-node/pull/1288) being merged.

now merged!

**Review by CodiePP — APPROVED** [review]
tested it with _shelley3pools_

**Pull request merged**